src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+const liArray = [...document.querySelectorAll('li')];
+const liEl = liArray.filter((li) => li.querySelector('ul'));
+
+liEl.forEach((li) => {
+  const span = document.createElement('span');
+
+  span.textContent = li.firstChild.textContent;
+  li.firstChild.remove();
+  li.prepend(span);
+});
+
+tree.addEventListener('click', (e) => {
+  if (e.target.tagName === 'SPAN') {
+    const parentLi = e.target.parentElement;
+    const subList = parentLi.querySelector('ul');
+
+    if (subList) {
+      subList.style.display =
+        subList.style.display === 'none' || '' ? 'block' : 'none';
```

Comment:
There is a logical error in this line. The condition `subList.style.display === 'none' || ''` is incorrect. The empty string `''` should not be part of the condition. Instead, you should check if `subList.style.display` is either 'none' or not set (empty string), which can be done using `!subList.style.display`.